targets/targets.json

```diff
@@ -2740,6 +2740,58 @@
         ],
         "device_name": "STM32G071RBTx",
         "bootloader_supported": true
+    },
+	"NUCLEO_G031K8": {
+        "inherits": [
+            "MCU_STM32_BAREMETAL"
+        ],
+        "supported_form_factors": [
+            "ARDUINO",
+            "MORPHO"
+        ],
+        "core": "Cortex-M0+",
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI",
+                "value": "USE_PLL_HSI",
+                "macro_name": "CLOCK_SOURCE"
+            },
+            "lpticker_lptim": {
+                "help": "This target supports LPTIM. Set value 1 to use LPTIM for LPTICKER, or 0 to use RTC wakeup timer",
+                "value": 1
+            },
+            "hse_value": {
+                "help": "HSE default value is 25MHz in HAL",
+                "value": "8000000",
+                "macro_name": "HSE_VALUE"
+            }
```

Comment:
Not useful for that board ?

Reply:
Yes that chip does not appear to have an HSE crystal input

Reply:
Resolved with c3a59a253b48d54c3ce383f87c87d0ba42ad910d